lib/sidekiq_fast_debounce/middleware/client.rb

```diff
@@ -18,26 +18,21 @@ class FastDebounce
         #   the job is not to be enqueued into redis, otherwise the block's
         #   return value is returned
         # @yield the next middleware in the chain or the enqueuing of the job
-        def call(worker_class, job, _queue, _redis_pool)
+        def call(_worker_class, job, _queue, _redis_pool)
```

Comment:
the class description has a typo but I cannot explicitly mark it because it's not in the diff